zsh/functions/g

```diff
@@ -1,8 +1,8 @@
 # No arguments: `git status`
 # With arguments: acts like `git`
 g() {
-  if [[ $# > 0 ]]; then
```

Comment:
I was wrong: http://zsh.sourceforge.net/Doc/Release/Conditional-Expressions.html

`>` is ASCII comparison ("z" > "a") and `-gt` is numeric comparison.